Workbooks/Virtual Machines - Performance Analysis/Performance Analysis for a Group of VMs (Dev)/Performance Analysis for a Group of VMs (Dev).workbook

```diff
@@ -433,435 +325,192 @@
+        "query": "// {Test} {Counter:label}\r\n{StandardQuery}",
```

Comment:
is this "standardquery" extra param that formats a giant query still necessary? it can't be just done in the query text of the step itself?  why not?

also, it has 2 parameters in a comment, why?  it will cause this query to be invalidated and run whenever those prams change too.  aren't they used in the thing that generated the query in the first place?